addon/serializers/node.js

```diff
@@ -1,4 +1,26 @@
 import OsfSerializer from './osf-serializer';
 
 export default OsfSerializer.extend({
+    serialize(snapshot) {
+        // Normal OSF serializer strips out relationships. We need to add back primaryFile/node/provider for this endpoint
+        const res = this._super(...arguments);
+        res.data.relationships = {};
+        let hasRelation = false;
+        for (var rel in snapshot.record._dirtyRelationships) {
+            let relationship = Ember.String.underscore(rel);
+            if (relationship.includes('license')) {
```

Comment:
Good catch on this one - I completely forgot we were stripping relationships out.